Dockerfile-csi-controller.test

```diff
@@ -17,18 +17,24 @@
 #    Its similar to the Dockerfile, but with additional requirements-tests.txt and ENTRYPOINT to run the local tests.
 
 FROM registry.access.redhat.com/ubi8/python-38:1-75.1638364053 as builder
-COPY controller/requirements.txt /tmp/
 USER root
 RUN if [[ "$(uname -m)" != "x86"* ]]; then yum install -y rust-toolset; fi
 USER default
 RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
 # avoid default boringssl lib, since it does not support z systems
 ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
```

Comment:
I think these two lines should come right before the `pip3 install`

Reply:
done